app/app_config.go

```diff
@@ -207,7 +207,6 @@ var (
 		minttypes.ModuleName,
 		stakingtypes.BondedPoolName,
 		stakingtypes.NotBondedPoolName,
-		tiertypes.ModuleName,
 		tiertypes.DeveloperPoolName,
 		tiertypes.InsurancePoolName,
 		// We allow the following module accounts to receive funds:
```

Comment:
just for clarity, can you move the removed line to the comment?

```suggestion
		// We allow the following module accounts to receive funds:
		// govtypes.ModuleName
		// tiertypes.ModuleName
```